sourcegraph/sourcegraph
# Configure git-lfs for *.har files

Configures `git-lfs` to manage browser extension integration tests snapshots. 
Motivation: see [this thread](https://sourcegraph.slack.com/archives/C01N83PS4TU/p1645537733042669).

## Test plan
- run `sg test bext-build`
- update integration tests snapshots `yarn --cwd client/browser record-integration`
- add updated snapshots to git index and commit
- ensure that snapshot tests on remote include reference to Git LFS instead of actual content
